### PR TITLE
Created Creatable.Safe types for Authorization and Verification

### PR DESCRIPTION
**Pull request by nilssonemma**
- [x] https://github.com/intergiro/model-card/pull/197

### CODE REVIEW

File: Authorization/Creatable/Safe/Card.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import * as authly from "authly"
+import * as model from "@payfunc/model-card"
+
+export interface Card {
+	iin: string
+	last4: string
+	scheme: model.Card.Scheme
+	token: authly.Token
+	expires: model.Card.Expires
+	csc?: "matched" | "mismatched" | "present"
```
Comment (robinnedergard) [inline Authorization/Creatable/Safe/Card.ts:11]: Where are we supposed to put the "magic string" with the token if they have same property names?

**Review by robinnedergard — CHANGES_REQUESTED** [review]

**Review by asnahassan — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -56,7 +56,7 @@
   },
   "dependencies": {
     "@payfunc/model-base": "0.1.34",
-    "@payfunc/model-card": "0.2.40",
+    "@payfunc/model-card": "0.2.42",
```
Comment (Magnus93) [inline package.json:59]: There is a newer version
```suggestion
    "@payfunc/model-card": "0.2.43",
```

File: Authorization/Creatable/index.ts [hunk 3]
```diff
@@ -105,53 +78,49 @@ export namespace Creatable {
...
+				card: { ...card, verification: verification ?? card.verification },
+				currency: value.currency,
+				items: value.items ?? value.amount ?? 0,
+				browser: value.browser,
+				category: value.category,
+				contact: value.contact,
+				recurring: value.recurring,
+				target: value.target,
+			}
+		return result // Todo: Clean up function
```
Comment (Magnus93) [inline Authorization/Creatable/index.ts:124]: I did not found any way you can change this without making it more complicated. Typescript is just stupid sometimes.

**Remove comment**

File: Verification/Creatable/Base.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+	recurring?: Authorization.Recurring
+	contact?: base.Contact
+	target?: string
+	category?: "purchase" | "withdrawal"
+}
+export namespace Base {
+	export function is(value: Base | any): value is Base {
+		return (
+			typeof value == "object" &&
+			typeof value.number == "string" &&
```
Comment (Magnus93) [inline Verification/Creatable/Base.ts:23]:
```suggestion
			typeof value == "object" && 
			value &&
			typeof value.number == "string" &&
```

File: Verification/Creatable/Base.ts [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+			(typeof value.items == "number" ||
+				base.Item.is(value.items) ||
+				(Array.isArray(value.items) && value.items.every(base.Item.is))) &&
+			(value.response == undefined ||
+				(typeof value.response == "object" &&
+					typeof value.response.type == "string" &&
```
Comment (Magnus93) [inline Verification/Creatable/Base.ts:29]:
```suggestion
				(typeof value.response == "object" &&
				value.response &&
					typeof value.response.type == "string" &&
```

File: Authorization/Creatable/Base.ts [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+	category?: "purchase" | "withdrawal"
+	browser?: base.Browser
+	contact?: base.Contact
+	target?: string
+}
+export namespace Base {
+	export function is(value: Base | any): value is Base {
+		return (
+			typeof value == "object" &&
+			(value.id == undefined || authly.Identifier.is(value.id, 16)) &&
```
Comment (Magnus93) [inline Authorization/Creatable/Base.ts:24]:
```suggestion
			typeof value == "object" &&
			value &&
			(value.id == undefined || authly.Identifier.is(value.id, 16)) &&
```

**Review by Magnus93 — CHANGES_REQUESTED** [review]

**Pull request merged**